### PR TITLE
Pull request

### PR DESCRIPTION
**Pull request by weibinyu**

### CODE REVIEW

File: src/data/members.js [hunk 1]
```diff
@@ -34,6 +34,7 @@ var members = {
 	as223my: require("./users/as223my.json"),
 	mattiaslj: require("./users/mattiaslj.json"),
 	ea222pu: require("./users/ea222pu.json")
```
Comment (krawaller) [inline src/data/members.js]: You must add a comma to the line above the one you added!

Comment (weibinyu) [inline src/data/members.js]: o sorry i forgot...

**Comment by weibinyu** [comment]
is it ok now?

**Comment by krawaller** [comment]
Looks good! Welcome! :)

**Pull request merged**